src/main/java/com/travelwith/servertravelwith/user/service/UserService.java

```diff
@@ -6,21 +6,46 @@
 
 package com.travelwith.servertravelwith.user.service;
 
+import com.travelwith.servertravelwith.user.model.Role;
 import com.travelwith.servertravelwith.user.model.UserCreationStatus;
 import com.travelwith.servertravelwith.user.model.UserEntity;
 import com.travelwith.servertravelwith.user.model.UserRest;
 import com.travelwith.servertravelwith.user.repository.UserRepository;
 import lombok.extern.slf4j.Slf4j;
 
+import javax.annotation.PostConstruct;
+import java.util.Optional;
+
 @Slf4j
 public class UserService {
 
+    private static final String ANDROID_CLIENT_USERNAME = "AndroidClient";
+    private static final String ANDROID_CLIENT_PASSWORD = "s6NY&<u)<asP+nUr";
+    private static final String ANDROID_CLIENT_EMAIL = "this@is.email";
```

Comment:
lol po co mail ?

Reply:
mamy not nulla panie a ten uzytkownik laduje w bazie :D

Reply:
To może to nie jest User ;) tylko jakiś Account albo coś 😃

Reply:
Dobra nie bede cisnął

Reply:
sugerujesz zeby go wgl nie wrzucac do bazki ? mozna tak zrobic